asset/asset.go

```
@@ -0,0 +1,241 @@
+//go:generate go run ./tools/gqlgen/main.go
+
+import (
+	"time"
+
+	"github.com/reearth/reearthx/account/accountdomain"
+)
+
+type Asset struct {
+	id                      AssetID
+	groupID                 *GroupID // projectID in visualizer and cms, workspaceID in flow
+	workspaceID             *WorkspaceID
```

golangci-lint (1.64.8)

field `workspaceID` is unused

(unused)